src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -7,6 +7,9 @@
 use Wallabag\CoreBundle\Entity\Entry;
 use Wallabag\CoreBundle\Tools\Utils;
 use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeExtensionGuesser;
+use Symfony\Component\Validator\Constraints\Language as LanguageConstraint;
```

Comment:
Language is `en` and not `en_GB` for instance. However, the OpenGraph `og:locale` format is `language_TERRITORY`, so it should be `Symfony\Component\Validator\Constraints\Locale` instead.

Handling merging `en_GB` and `en_US` into `en` should be done somewhere else.